pandas/_libs/tslibs/offsets.pyx

```diff
@@ -2779,16 +2779,34 @@ cdef class Week(SingleConstructorOffset):
     """
     Weekly offset.
 
+    This can increment the date object by given week or weekdays.
     Parameters
     ----------
     weekday : int or None, default None
-        Always generate specific day of week. 0 for Monday.
+        Always generate specific day of week. 0 for Monday and 6 for Sunday.
 
-    Examples
+    See Also
     --------
-    >>> ts = pd.Timestamp(2022, 1, 1)
-    >>> ts + pd.offsets.Week()
-    Timestamp('2022-01-08 00:00:00')
+    pd.tseries.offsets.WeekOfMonth : Describes monthly dates like "the Tuesday of the 2nd week of each month".
+
+    Examples
+    ---------
+    >>> date_format = "%Y-%m-%d"
+    >>> date_object = pd.to_datetime("2023-01-13",format = date_format)
+    >>> print(date_object)
```

Comment:
you don't need `print`, just something like
```
>>> date_object
'2023-01-13 00:00:00'
```
(I haven't tried running this, so it may not be exactly correct)